cmd/crc/cmd/delete.go

```diff
@@ -34,6 +34,10 @@ func runDelete(arguments []string) {
 	deleteConfig := machine.DeleteConfig{
 		Name: constants.DefaultName,
 	}
+
+	// Check if the crc instance exists
+	exitIfMachineMissing(deleteConfig.Name)
```

Comment:
The comment is now redundant as you have a very explicit function name

Reply:
@cfergeau Done